core/src/main/java/io/grpc/internal/AbstractClientStream.java

```diff
@@ -193,6 +196,8 @@ public final boolean isReady() {
   protected abstract static class TransportState extends AbstractStream.TransportState {
     /** Whether listener.closed() has been called. */
     private final StatsTraceContext statsTraceCtx;
+    // TODO(zpencer): remove @Nullable when okhttp supports transport tracing
+    @Nullable private final TransportTracer transportTracer;
```

Comment:
Provide a getter for this, to prevent needing the field in subclasses?

Reply:
The superclass (`AbstractClientStream.TransportState`) does not need it, it's only passing it down to the Deframer. Would we want to punch a hole all the way into the deframer to get this value?

Reply:
I was thinking just having the getter here, for use by Netty+OkHttp.

Reply:
Ah understood, moving it.